android/src/main/java/com/github/wuxudong/rncharts/charts/ChartBaseManager.java

```diff
@@ -405,9 +406,20 @@ public void setData(Chart chart, ReadableMap propMap) {
      */
     @ReactProp(name = "highlightValue")
     public void setHighlightValue(Chart chart, ReadableMap propMap) {
+        Highlight h = new Highlight(
+            (float) propMap.getDouble("x"), 
+            Float.NaN, // y
+            propMap.getInt("dataSetIndex")
+        );
+
+        // dataIndex required for CombinedChart to pin point a highlight position
+        Integer dataIndex = propMap.getInt("dataIndex");
```

Comment:
I think if `dataIndex` is null, you'll actually hit this error:
```
com.facebook.react.bridge.UnexpectedNativeTypeException: TypeError: expected dynamic type `int64', but had type `null'
```

Can instead do:
```
Integer dataIndex = propMap.hasKey("dataIndex") && !propMap.isNull("dataIndex") ? propMap.getInt("dataIndex") : null;
```

Reply:
good catch! updated.